src/event-stream.ts

```diff
@@ -0,0 +1,362 @@
+/**
+ * @fileoverview Event stream backed by the events API
+ */
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+import { Promise } from 'bluebird';
+import qs from 'querystring';
+import { Readable } from 'stream';
+import util from 'util';
```

Comment:
Looks like `util` is no longer used in this file

Reply:
Thanks! Will fix in a follow-up.